src/index.ts

```diff
@@ -101,9 +102,16 @@ const getNamedType = (
                             foundType.types && foundType.types[0],
                         );
                     case 'scalar':
-                        // it's a scalar, let's use a string as a value.
-                        // This could be improved with a custom scalar definition in the config
-                        return `'${casual.word}'`;
+                        // it's a scalar, let's use a string as a value if there is no custom
+                        // mapping for this particular scalar
+                        if (!customScalars || !customScalars[foundType.name]) {
+                            return `'${casual.word}'`;
+                        }
+
+                        // If there is a mapping to a `casual` type, then use this
+                        const embeddedGenerator = casual[customScalars[foundType.name]];
```

Comment:
What if there is no mapping? This would return `undefined`
Maybe throw an exception when there is no mapping?

A future improvement would be to support custom functions from the config, but I think this is a good first step.
We'll see if people need custom functions in a future issue/PR

Reply:
If you scroll up a bit above, you will see

```tsx

  // it's a scalar, let's use a string as a value if there is no custom
   // mapping for this particular scalar
   if (!customScalars || !customScalars[foundType.name]) {
      return `'${casual.word}'`;
    }
```

so I'm defaulting to a word if there is no scalar. Wouldn't that suffice?